docs/docsite/rst/guide_aci.rst

```diff
@@ -0,0 +1,298 @@
+
+Click the name of the user you want to add a certificate to, in the **User Certificates** area, click the **+** sign and in the **Create X509 Certificate** enter a certificate name in the **Name** field (If you use the basename of your private key here, you don't need to enter **certificate_name** in Ansible) and copy&paste your X.509 certificate in the **Data** field.
+
+Obviously you can automate this by using the following Ansible task:
+
+.. code-block:: yaml
+
+    - name: Ensure we have a certificate installed
+      aci_aaa_user_certificate:
+        host: my-apic-1
+        username: admin
+        password: my-password
+    
+        aaa_user: admin
+        certificate_name: admin
+        certificate: "{{ lookup('file', 'pki/admin.crt') }}"  # This wil read the certificate data from a local file
+
+**NOTE:** Signature-based authentication only works with local users.
+
+
+Use Signature-based Authentication with Ansible
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+ You need the following parameters with your ACI module(s) for it to work:
+
+.. code-block:: yaml
+
+    username: admin
+    private_key: pki/admin.key
+    certificate_name: admin  # This could be left out !
+
+**NOTE:** If you use a certificate name in ACI that matches the private key's basename, you can leave out the `certificate_name` parameter like the example above.
+
+More information
+,,,,,,,,,,,,,,,,
+More information about Signature-based Authentication is available from `Cisco APIC Signature-Based Transactions <https://www.cisco.com/c/en/us/td/docs/switches/datacenter/aci/apic/sw/kb/b_KB_Signature_Based_Transactions.html>`_.
+
+Using ACI REST with Ansible
+---------------------------
+While already a lot of ACI modules exists in the Ansible distribution, and the most common actions can be performed with these existing modules, there's always something that may not be possible with off-the-shelf modules.
+
+The **aci_rest** module provides you with direct access to the APIC REST API and enables you to perform any task not already covered by the existing modules. This may seem like a complex undertaking, but you can generate the needed REST payload for any action performed in the ACI web interface effortless.
+
+Using the aci-rest module
+.........................
+The **aci_rest** module accepts the native XML and JSON payloads, but additionally accepts inline YAML payload (structured like JSON). The XML payload requires you to use a path ending with `.xml` whereas JSON or YAML require path to end with `.json`.
+
+When you're making modifications, you can use the POST or DELETE methods, whereas doing just queries require the GET method.
+
+For instance, if you would like to ensure a specific tenant exists on ACI, these below four examples are identical:
+
+**XML** (native)
+
+.. code-block:: yaml
+
+    - aci_rest:
```

Comment:
`- name: `

Reply:
Well, all these tasks are doing the same thing, so I wanted to show only what is needed/relevant.
That's why I deliberately left out the name-key.